src/test/java/uk/gov/hmcts/darts/arm/api/impl/ArmDataManagementApiImplTest.java

```diff
@@ -97,4 +103,15 @@ void getResponseBlobData() {
         BinaryData binaryData = armDataManagementApi.getBlobData(blobNameAndPath);
         assertEquals(data, binaryData);
     }
+
+    @Test
+    void downloadArmData() {
+
+        var inputStream = ByteArrayInputStream.nullInputStream();
```

Comment:
not the playlist.xml Michael. 
ARM will store audio files for now, but in the future it could store pdf, etc... so we made the download operation capable of dealing with any type of binary file